provider/postgis/util.go

```diff
@@ -50,20 +50,13 @@ func genSQL(l *Layer, pool *pgx.ConnPool, tblname string, flds []string) (sql st
 		}
 	}
 
-	for i := range flds {
```

Comment:
I think this code was there to wrap all the SELECT values in quotes to avoid issues when the field is also a postgres keyword (i.e. name -> "name"). I don't recall if we have a test for this, but I think we should double-check before we purge this code. Looks like some comments would be helpful here.

Reply:
The diff is messed up, and it's best seen side-by-side. If you look at line 55 and 59 of the new file the looping and adding quotes is still there. In the old file, it was looping twice and doing checks against the new (quoted) fields